source/jquery.flot.js

```diff
@@ -169,6 +169,7 @@ Licensed under the MIT license.
                         show: false,
                         lineWidth: 2, // in pixels
                         barWidth: 1, // in units of the x axis
+                        absoluteBarWidth: false,
```

Comment:
We should employ another pattern for expanding the functionality of the 'barWidth' property. See the 'tickSize' option in the xaxis set of options. You can give it either a number or a tuple of values, where one of the values is the unit. We should adopt that here. To be clear, that means getting rid of the 'absoluteBarWidth' property, and just have that be the extra tuple value of 'barWidth'.